src/api/getTrendingArticles.js

```diff
@@ -13,6 +13,7 @@ export const getTrendingArticles = (lang, country) => {
   return cachedFetch(url, data => {
     const page = data.query.pages[0]
     if (page.missing) {
+      sendErrorLog({ message: page, url })
```

Comment:
the page is an object, the message here should be `string`

Reply:
ah, that's fine as it will stringify later

Reply:
Yea, does it make sense to pass in the whole `page` object though?

Reply:
Actually answering my own question, the `400` error seems to be related to `message` not being a string